.github/workflows/release.yml

```diff
@@ -32,37 +26,22 @@ jobs:
         XDP_TEST_IN_CI: 1
 
     steps:
-      - name: Configure git user
+      - name: Configure environment
         run: |
-          git config --global user.name "github-actions[bot]"
-          git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
           git config --global --add safe.directory $GITHUB_WORKSPACE
 
       - name: Checkout the repository
         uses: actions/checkout@v4
 
-      - name: Update translation files
-        run: |
-          meson setup . _build
-          meson compile -C _build/ xdg-desktop-portal-update-po
-          git add po/*po
-          git commit -m "Update po files"
-          git clean -fxd
-
       - name: Build xdg-desktop-portal
         run: |
           meson setup . _build
           meson dist -C _build
 
       - name: Extract release information
-        env:
-          releaseVersion: ${{ github.event.inputs.version }}
-          preRelease: ${{ github.event.inputs.preRelease }}
         run: |
           # Extract the release version
-          if [ -z $releaseVersion ]; then
-            releaseVersion=`perl -0777nE 'print $& if /(?<=Changes in ).*/' NEWS.md`
-          fi
+          releaseVersion=`perl -0777nE 'print $& if /(?<=Changes in ).*/' NEWS.md`
```

Comment:
Given you've already built it in the previous step anyway it may be simpler to do:
```console
$ meson introspect --projectinfo build/ | jq -r .version
1.19.3
```